src/atoms/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './hooks';
+export * as style from './style';
+export type * from './types';
+
```

Comment:
Should this file have `export * from './icons';`?

Reply:
Yep, adding it now

Reply:
I think it should be just `./icons` and not `./icons/AmplifyIcons` in case we add other icon files c: